prac_08/squaring.kv

```diff
@@ -0,0 +1,20 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_number
+            text: ' '
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(int(input_number.text))
+        Label:
+            id: output_label
+            font_size: 48
+            color: (0, 1, 1, 1)
```

Comment:
I think we are supposed to give out in a pink color, I am not sure, but I think this is not pink color